tools/cockpit.spec

```diff
@@ -131,7 +131,7 @@ embed or extend Cockpit.
 
 %package pcp
 Summary: Cockpit PCP integration
-Requires: %{name}-bridge = %{version}-%{release}
```

Comment:
This requires bridge because it's not a js package. But a binary that needs to work with bridge.